src/modules/inspect/inspect-correlation/components/inspect-panel/components/correlation-list/correlation-list.ts

```diff
@@ -11,16 +11,28 @@ import {
 } from '../../../../../../../contracts/index';
 import environment from '../../../../../../../environment';
 import {getBeautifiedDate} from '../../../../../../../services/date-service/date.service';
-import {IProcessInstanceWithCorrelation} from '../../../../contracts/index';
 
+const PAGE_SIZES = [20, 50, 100, 200];
+const MIN_PAGESIZE = PAGE_SIZES[0];
+export const DEFAULT_PAGESIZE = PAGE_SIZES[1];
+
+const PAGINATION_SIZE = 10;
```

Comment:
Nichts schlimmes nur verstehe ich das spacing hier nicht:
Zwischen den verschiedenen Konstanten machst du eine Newline aber zwischen inject und einer Konstante nicht?

Reply:
Ich wollte mit der Leerzeile verdeutlichen das die `PAGINATION_SIZE` nichts mit der PageSize zutun hat.

```suggestion
const PAGINATION_SIZE = 10;

```

Reply:
Ja das verstehe ich aber die `PAGINATION_SIZE` hat ja noch weniger mit dem `inject` zu tun 😉